glslc/README.asciidoc

```diff
@@ -252,6 +252,26 @@ code, such as identifier names and line numbers.
 NOTE: Currently this option has no effect.  Full functionality depends on
 glslang support for generating debug info.
 
+==== `-masm=<dialect>`
+
+Select dialect for compilation output. Supported options are listed in
```

Comment:
I expected -masm=dialect to only take effect when you're already selecting assembly output with -S.
That follows the example of GCC.
https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html#index-masm_003d_0040var_007bdialect_007d-2837

And in that case,  you should be able to select between c-init-list or spv.